pelorus-operator/bundle/manifests/charts.pelorus.konveyor.io_pelorus.yaml

```diff
@@ -15,24 +15,56 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: Pelorus is the Schema for the pelorus API
+        description: Pelorus is the Schema for pelorus API to adopt instance to the
+          requested workflow
         properties:
           apiVersion:
             description: 'APIVersion defines the versioned schema of this representation
               of an object. Servers should convert recognized schemas to the latest
               internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: 'Kind is a string value representing the REST resource this
+            description: |-
+              Kind is a string value representing the REST resource this
               object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```

Comment:
would be better if we rephrase  to something like "Kind is a string value, **in CamelCase**, representing the REST resource this..."?

Reply:
This one comes from operator-sdk, so you can file issue to their project.